Source/Dafny/DafnyOptions.cs

```diff
@@ -103,6 +103,9 @@ public enum IncludesModes { None, Immediate, Transitive }
     public bool WarningsAsErrors = false;
     [CanBeNull] private TestGenerationOptions testGenOptions = null;
     public bool ExtractCounterexample = false;
+    public string VerificationLoggerConfig = null;
+    // Working around the fact that xmlFilename is private
+    public string BoogieXmlFilename = null;
```

Comment:
Do you want to cut an Issue or PR with Boogie to get this changed?

Reply:
Not really, actually. Using the `/xml` option is a decent solution but a bit hacky. I'd rather follow up with a more structured event model for Boogie to communicate results to clients, something like the `OutputPrinter` interface but with more than just string messages.